source/base/data_out_base.cc

```diff
@@ -8825,7 +8825,11 @@ DataOutBase::write_hdf5_parallel(
   (void)mesh_filename;
   (void)solution_filename;
   (void)comm;
-  AssertThrow(false, ExcMessage("HDF5 support is disabled."));
+  AssertThrow(false,
```

Comment:
Would it be possible to do it like this: https://github.com/dealii/dealii/blob/6dff4a664823d2a98a46d8ab2e5d4e4ec7a3b439/include/deal.II/base/exceptions.h#L1103-L1107?